test/unit/spec/appWidgets/input/checkboxInputSpec.js

```diff
@@ -105,19 +106,5 @@ define([
                 input.dispatchEvent(new Event('change'));
             });
         });
-
-        it('should show message when configured', (done) => {
```

Comment:
The `showOwnMessages` feature is not implemented in the checkboxInput (it does appear in other input types, though).